web/src/styles/global.css

```
@@ -25,6 +25,26 @@
   unicode-range: U+0100-02BA, U+02BD-02C5, U+02C7-02CC, U+02CE-02D7, U+02DD-02FF, U+0304, U+0308, U+0329, U+1D00-1DBF, U+1E00-1E9F, U+1EF2-1EFF, U+2020, U+20A0-20AB, U+20AD-20C0, U+2113, U+2C60-2C7F, U+A720-A7FF;
 }
 
+/* ---------- Self-hosted Fraunces font (Public Notice display face) ----------
+   Served from /public/fonts, same self-hosting rationale as Inter above.
+   Display/headline roles only (regular for body-adjacent display copy, 600 for
+   semibold headlines) — body text never uses Fraunces. Latin subset only. */
+@font-face {
+  font-family: 'Fraunces';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)